docs/v1/models.md

```diff
@@ -17,9 +17,9 @@
 ### Transaction
 + hash: `ghf6msRhE4jRf5DPib9UHD1msadvmZs9o53V9FQTb11` (string,required)
 + source: `GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ` (string,required) 
-+ fee: `10000` (string,required)
++ fee: `10000` (string,required) - when user sent money to another account, fee have to be charged. 
```

Comment:
`money` -> `boscoin`
`fee` is already charged for that amount

Reply:
ok.